napari_animation/_tests/fixtures.py

```diff
@@ -1,26 +1,11 @@
 import pytest
+from napari.utils._testsupport import make_napari_viewer  # noqa: F401
```

Comment:
Love it!

Reply:
Here too, I'm not sure whether this might cause you issues.  Fixtures, for better or worse, are not designed to be imported explicitly.  If you have napari installed, pytest will find this fixture (import not necessary).  Because of how terribly ugly that is, we tend to add a comment next to where the fixture is being used, explaining where it comes from, but not actually importing it

Reply:
looks like this will be fixed in #42, thanks for the tip!